src/devtools/devtools.helper.js

```diff
@@ -0,0 +1,22 @@
+export function removeCyclicReferences(apps) {
+  // we must get rid of cyclic references before we can return apps -- otherwise, devtools gets an error because JSON.stringify doesn't resolve cyclic references
+  return apps.map(app => {
+    if(Object.keys(app.parcels).length > 0) {
+      app.parcels = Object.keys(app.parcels).map(parcKey => {
+        return removeParcelCyclicReferences(app.parcels[parcKey])
+      })
+    }
+    return app
+  })
+}
```

Comment:
I might be wrong here, IIRC because app is an object it's passed by reference. Meaning this delete methods are going to modify the actual app instead of modifying a duplicate of the app